sdk/samples/router/general.py

```diff
@@ -0,0 +1,320 @@
+        Route(
+            id='baaa',
+            endpoint='http://fox-says.baa',
+            timeout='20ms'
+        ),
+        Route(
+            id='oink',
+            endpoint='http://fox-says.oink',
+            timeout='20ms'
+        ),
+        Route(
+            id='ring-ding-ding',
+            endpoint='http://fox-says.ring-ding-ding',
+            timeout='20ms'
+        )
+    ]
+
+    # Create some traffic rules
+    rules = [
+        TrafficRule(
+            conditions=[
+                HeaderTrafficRuleCondition(
+                    field='name',
+                    values=['cat']
+                )
+            ],
+            routes=[
+                'meow'
+            ]
+        ),
+        TrafficRule(
+            conditions=[
+                HeaderTrafficRuleCondition(
+                    field='name',
+                    values=['dog']
+                )
+            ],
+            routes=[
+                'woof'
+            ]
+        ),
+        TrafficRule(
+            conditions=[
+                HeaderTrafficRuleCondition(
+                    field='name',
+                    values=['sheep']
+                )
+            ],
+            routes=[
+                'baaa'
+            ]
+        ),
+        TrafficRule(
+            conditions=[
+                HeaderTrafficRuleCondition(
+                    field='name',
+                    values=['pig']
+                )
+            ],
+            routes=[
+                'oink'
+            ]
+        ),
+        TrafficRule(
+            conditions=[
+                PayloadTrafficRuleCondition(
+                    field='body',
+                    values=['sus']
+                )
+            ],
+            routes=[
+                'meow',
+                'woof',
+                'baaa',
+                'oink',
+                'ring-ding-ding'
+            ]
+        )
+    ]
+
+    # Create an experiment config (
+    experiment_config = ExperimentConfig(
+        type="xp",
+        config={
+            'variables':
+                [
+                    {'name': 'farm_id', 'field': 'farm_id', 'field_source': 'header'},
+                    {'name': 'country_code', 'field': 'country', 'field_source': 'header'},
+                    {'name': 'latitude', 'field': 'farm_lat', 'field_source': 'header'},
+                    {'name': 'longitude', 'field': 'farm_long', 'field_source': 'header'}
+                ],
+            'project_id': 102
+        }
+    )
+
+    # Create a resource request config for the router
+    resource_request = ResourceRequest(
+        min_replica=0,
+        max_replica=2,
+        cpu_request="500m",
+        memory_request="512Mi"
+    )
+
+    # Create a log config for the router
+    log_config = LogConfig(
+        result_logger_type=ResultLoggerType.NOP
+    )
+
+    # Create an enricher for the router
+    enricher = Enricher(
+        image="asia.gcr.io/gods-dev/echo:1.0.2",
```

Comment:
That's on a private GCR. Please replace it with a publicly accessible image and keep a note to avoid adding any Gojek specific stuff to the OSS repo

Reply:
Ok got that; will keep that in mind in the future.